### ISSUE TITLE
vec.normalize() is incorrect if values have changed

### ISSUE THREAD

**Issue by brianpeiris** [quinton-ashley/q5.js#20]
Looks like q5's optimizations don't account for changes to a vector's components.

```js
var vec = new p5.Vector();

vec.x = 2; 
vec.normalize();
console.log(vec.x); // gives 1

vec.y = 1;
vec.normalize();
console.log(vec.x) // gives 0.707 in p5, but still 1 in q5
```

**Comment by quinton-ashley** [comment]
thanks for reporting! Fixed in v1.9.1

**Comment by brianpeiris** [comment]
Thanks!
